### PR TITLE
C5

### PR DESCRIPTION
**Pull request by henryabel**

### CODE REVIEW

File: app/scripts/Services/Message.js [hunk 1]
```diff
@@ -1,27 +1,21 @@
 (function () {
     function Message($firebaseArray) {
-        var tempRoomId1 = "-Kwx2shcvslekPFi-lyC";
-        var tempRoomId2 = "-Kwx2u7ZbwL2YwJpyDK4";
-        var tempRoomId3 = "-Kwx2wPJPoZIZWFXoxah";
+        //console.log("in Messages");
```
Comment (RyanBalfanz) [inline app/scripts/Services/Message.js]: I would get rid of this commented code.

File: app/scripts/Services/Message.js [hunk 2]
```diff
@@ -1,27 +1,21 @@
...
-                    , sentAt: new Date().getTime()
-                    , roomId: tempRoomId3
-                });
-            }
-            //create sample messages
-            /*for (var i = 0; i < 4; i++) {
-                Message.add();
-            }*/
+        Message.send = function (roomId, content) {
+            console.log("it works");
```
Comment (RyanBalfanz) [inline app/scripts/Services/Message.js]: Once this "works", get rid of this too.

File: app/scripts/Services/Message.js [hunk 3]
```diff
@@ -1,27 +1,21 @@
...
+            messages.$add({
+                username: "<USERNAME HERE>"
```
Comment (RyanBalfanz) [inline app/scripts/Services/Message.js]: Hopefully, this is being worked on in another feature branch.

File: app/scripts/Services/Room.js [hunk 4]
```diff
@@ -1,5 +1,6 @@
 (function () {
     function Room($firebaseArray) {
+        //console.log("in Room");
```
Comment (RyanBalfanz) [inline app/scripts/Services/Room.js]: Again, remove dead code.

File: app/scripts/app.js [hunk 5]
```diff
@@ -8,7 +8,9 @@
             url: '/'
             , controller: 'HomeCtrl as home'
             , templateUrl: 'templates/home.html'
+        }).state('cookies', {
```
Comment (RyanBalfanz) [inline app/scripts/app.js]: What's the purpose of this state? I don't think it offers anything useful. It doesn't even have a URL or a template.

File: app/scripts/app.js [hunk 6]
```diff
@@ -8,7 +8,9 @@
...
+            controller: 'BlocChatCookies as bcc'
```
Comment (RyanBalfanz) [inline app/scripts/app.js]: `BlocChatCookies` isn't a controller, so this doesn't make sense.

File: app/scripts/controllers/BlocChatCookies.js [hunk 7]
```diff
@@ -0,0 +1,14 @@
+(function () {
+    console.log("in cookies");
```
Comment (RyanBalfanz) [inline app/scripts/controllers/BlocChatCookies.js]: I'd remove this.

File: app/scripts/controllers/HomeCtrl.js [hunk 8]
```diff
@@ -13,7 +22,10 @@
         vm.setActiveRoom = function (room) {
             document.getElementById("roomTitle").innerHTML = room.title;
```
Comment (RyanBalfanz) [inline app/scripts/controllers/HomeCtrl.js:23]: Why not just bind to the `activeRoom.title` in the template?

**Review by RyanBalfanz — COMMENTED** [review]
@henryabel: Good start, I left some comments.

File: app/scripts/controllers/HomeCtrl.js (hunk 8 above)
Comment (henryabel) [reply app/scripts/controllers/HomeCtrl.js:23]: You're right, I should have used ng-model there.  It would have saved me a lot of coding.

**Pull request merged**